docs/ops/matrix/MatMul_1.md

```diff
@@ -12,29 +12,43 @@
 
 Before matrix multiplication, there is an implicit shape alignment for input arguments. It consists of the following steps:
 
-1. If rank of an input less than 2 it is unsqueezed to 2D tensor by adding axes with size 1 to the left of the shape. For example, if input has shape `[S]` it will be reshaped to `[1, S]`. It is applied for each input independently.
+1. Applying transpositions specified by optional `transpose_a` and `transpose_b` attributes for 2D tensors and bigger.
```

Comment:
Please, be more specific: (1) instead of "2D tensors and bigger" say that only two right-most dimensions are transposed/swapped, other dimensions are remain the same, and (2) say explicitly that transposes are ignored for 1D tensors, put it as a consequence of the rule (1),

Reply:
Done